x-pack/test_serverless/api_integration/test_suites/common/management/multiple_spaces_enabled.ts

```diff
@@ -26,10 +26,11 @@ import { FtrProviderContext } from '../../../ftr_provider_context';
 export default function ({ getService }: FtrProviderContext) {
   const svlCommonApi = getService('svlCommonApi');
   const roleScopedSupertest = getService('roleScopedSupertest');
-  let supertestWithAdminScope: SupertestWithRoleScopeType;
+  let supertestAdminWithApiKey: SupertestWithRoleScopeType;
+  let supertestAdminWithCookieCredentials: SupertestWithRoleScopeType;
```

Comment:
Splitting since some APIs are internal, others are public

Reply:
I left it as is for now. It sounds like you will update the test suite while working on that issue